README.md

```diff
@@ -1,21 +1,38 @@
 # Excessive Motion GUI
+
 This repository contains the default Excessive Motion UI frontend and a minimal CEF app for running the GUI on dedicated controller hardware.
 
 ### Project Structure
+
 - TODO: this (Webpack, TypeScript, ...)
 
-### Build Requirements
+The **Frontend** and **CEF App** projects are set up to work from the root reporitory directory, so no directory changes are required to build/run both.
+
+## Frontend [Webpack/TypeScript]
+
+### Build requirements
+
 - Node.js
+  - Install `nvm` by following the instructions here: https://github.com/nvm-sh/nvm#installing-and-updating
+- pnpm
+  - Install using `corepack`:
 
-## Development
-The **Frontend** and **CEF App** projects are set up to work from the root reporitory directory, so no directory changes are required to build/run both.
+```sh
+corepack enable
```

Comment:
`corepack enable` does not work if the user does not have permissions to write to /bin
Possible fixes:
1. `corepack enable --install-directory ~/bin` (`mkdir ~/bin` would be required for the command to work if ~/bin does not exist)
2. `sudo corepack enable`

Reply:
I've updated to include the non-sudo version in the readme, as well updated the `nvm` notes slightly to hopefully make them a bit clearer - using `nvm` also avoids the above issue.